src/Styleguide/Components/EntityHeader.tsx

```diff
@@ -35,7 +47,11 @@ export const EntityHeader: SFC<EntityHeaderProps> = ({
       )}
 
       <Flex flexDirection="column" justifyContent="center" width="100%">
-        <Serif size="3" weight="semibold" color="black100">
+        <Serif
+          size={nameFontSize ? nameFontSize : "3"}
+          weight={nameFontSize ? nameFontWeight : "semibold"}
```

Comment:
Are these configurations for repsonsiveness or does it just need be changed based on the context in which the component renders?

Reply:
@zephraph it's definitely based on context ([the QA ticket for context](https://artsyproduct.atlassian.net/browse/GROW-1014)). We were a little iffy about changing the component. I went ahead and opened the PR to get a sense of what we should be doing in these cases. 

We also thought that Owen might not be aware that is a component used globally which might have impacted he's decision to make these minor UI changes. Since Owen is out, @nicoleyeo do you have any input?

cc: @eessex

Reply:
Hi — hmmm I do not recommend altering "entity-header" here. It's visually similar but is a component with a very different use.

If I understand the context of this component correctly (a small representation of a collection), this should be called something like "entity-collection" and is actually more similar to "entity-artist" or "entity-gallery" which we have not formalized yet in Palette (or Reaction, really).

For simplicity, I'd recommend creating a new "CollectionEntity" (in Reaction) to be safe and avoid breaking other areas. @xtina-starr lmk if this needs further detail, happy to discuss.

(@zephraph note for us to formalize "Entity" components in Palette soon).

Question: Is there a specific reason why we are building this in Reaction and not in Palette?